tests/acceptance/test_literature_suggestion_form.py

```diff
@@ -48,12 +48,11 @@ def test_literature_create_thesis_manually(login):
         'abstract': 'Lorem ipsum dolor sit amet, consetetur sadipscing elitr.',
         'report-number-0': '100',
         'report-number-1': '101',
-        'supervisor': 'Mister Yellow',
-        'supervisor-affiliation': 'CERN',
-        'thesis-date': '2001-01-01',
-        'thesis-defense': '2001-01-01',
         'degree-type': 'bachelor',
+        'thesis-date': '2001-01-01',
         'institution': 'Wisconsin U., Madison',
+        'supervisor': 'Mister Yellow',
+        'supervisor-affiliation': 'CERN',
```

Comment:
Same comment as above: you are reordering for clarity (this is a dictionary, so the order does not matter!), so you should be precise about it.